data/test-cases/plugins/MS Teams sync/Slash commands/Show.md

```diff
@@ -0,0 +1,46 @@
+---
+# (Required) Ensure all values are filled up
+name: "Show clash command"
+status: Active
+priority: Normal
+folder: Slash commands
+authors: "@AayushChaudhary0001"
+team_ownership: 
+- Change Team Name
+priority_p1_to_p4: P2 - Core Functions (Do core functions work?)
+
+# (Optional)
+location: null
+component: null
+tags: []
+labels: []
+tested_by_contributor: null
+
+# (Optional) Test type and tools
+cypress: null
+detox: null
+mmctl: null
+playwright: null
+rainforest: []
+manual_test_environments: []
+
+# Do not change
+id: null
+key: null
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Run the `/msteams-sync show` command in the desired MM channel.
+
+**Step 2**
+
+1. Run the `/msteams-sync show` command in the desired MM channel when the channel is not linked already.
```

Comment:
```suggestion
1. Run the `/msteams-sync show` command in the desired MM channel when the channel is not linked.
```

Reply:
Here we are writing a different test case as "Step 2" in all the files which is misleading. We should use a title like "Test case 1", Test case 2".

Reply:
No we can't, as this is a standard process for all the plugins.